packages/ui/src/app/pages/Council/Council.tsx

```diff
@@ -101,3 +157,6 @@ const StatisticsStyle = styled(Statistics)`
     grid-template-columns: 1fr 1fr;
   }
 `
+const CustomBlockTimeWrapper = styled(BlockTimeWrapper)`
+  grid-row-gap: 2px;
```

Comment:
```suggestion
  grid-row-gap: 2px;
  min-width: 165px;
  justify-content: flex-start;
```
On my end it looks bad, because the term started in `CET` and ends in `CEST` time:
![image](https://github.com/user-attachments/assets/b34c2c74-1647-426b-b98e-3e43c2a5b607)

After this fix it's more consistent:
![image](https://github.com/user-attachments/assets/5f01dcb6-c689-469a-a1f3-3c78719b80c4)